TestHelpers.Tests/TestHelpers.Tests.ncrunchproject

```diff
@@ -1,20 +0,0 @@
-<ProjectConfiguration>
```

Comment:
@jpreese @tathamoddie Do you use NCrunch?

Reply:
I do, yes. Though I'm not convinced it needs to be in source control.

Reply:
If we're going to remove these I think it makes sense to update the `.gitignore` to ensure they don't accidentally work their way back into the repro.

It looks like we already have a section for NCrunch but I don't know why this line didn't cause these files to be ignored:
https://github.com/System-IO-Abstractions/System.IO.Abstractions/blob/e6ec3d48138fbf708c593953b3a395c23f28a84b/.gitignore#L75

Maybe that was updated after these files were already added?

Reply:
Ah, that’s why Git wants to remove them :)